Test.java

```diff
@@ -0,0 +1,164 @@
+
+public class Test {
+
+    static Team galatasaray = new Team(1, "galatasaray", "GS");
+    static Team fenerbahce = new Team(2, "fenerbahce", "FB");
+    static Team besiktas = new Team(3, "besiktas", "BJ");
+    static Team sivasspor = new Team(4, "sivasspor", "SS");
+    static Team trabzonspor = new Team(5, "trabzonspor", "TS");
+    static Team göztepe = new Team(6, "göztepe", "GT");
+    static Team antalyaspor = new Team(7, "antalyaspor", "AS");
+    static Team gaziantepspor = new Team(8, "gaziantepspor", "GA");
+    static Team genclerbirligi = new Team(9, "genclerbirligi", "GB");
+    static Team bursaspor = new Team(10, "bursaspor", "BS");
+
+    public static void main(String[] args) {
+
+        TeamArray teamArr = new TeamArray();
+        MatchArray matchArr = new MatchArray();
+        Test t = new Test();
+
+        fillTeamArr(teamArr);
+        matchArr.fillMatchArr();
+        t.points(teamArr, matchArr);
+        teamArr.displayTeams();
+        matchArr.displayMatchArr();
+        teamArr.displayPoints();
+        champ(teamArr, matchArr);
+
+    }
+
+    public static void fillTeamArr(TeamArray teamArray) {
+        teamArray.teamArr[0] = galatasaray;
+        teamArray.teamArr[1] = fenerbahce;
+        teamArray.teamArr[2] = besiktas;
+        teamArray.teamArr[3] = sivasspor;
+        teamArray.teamArr[4] = trabzonspor;
+        teamArray.teamArr[5] = göztepe;
+        teamArray.teamArr[6] = antalyaspor;
+        teamArray.teamArr[7] = gaziantepspor;
+        teamArray.teamArr[8] = genclerbirligi;
+        teamArray.teamArr[9] = bursaspor;
+    }
+
+    public void points(TeamArray teamObj, MatchArray matchObj) {
+
+        for (int i = 0; i < teamObj.length; i++) {
+            for (int j = 0; j < teamObj.length; j++) {
+                if (i < j) {
+                    if (matchObj.matchArr[i][j] > matchObj.matchArr[j][i]) {
+                        teamObj.teamArr[i].wonMatch();
+                        teamObj.teamArr[j].lostMatch();
+                    } else if (matchObj.matchArr[i][j] == matchObj.matchArr[j][i]) {
```

Comment:
matchObj.matchArr[i][j] ve matchObj.matchArr[j][i] iki kere kullanıldığı için ve tam olarak neyi temsil ettiğini anlamak zor olduğu için bunlar için değişken oluşturabiliriz. örnek; homeGoals & awayGoals